dask_cuda/__init__.py

```diff
@@ -9,6 +31,33 @@
 import dask.dataframe.shuffle
 import dask.dataframe.multi
 import dask.bag.core
+import os
+
+
+def is_amd_gpu_available():
```

Comment:
@younseojava  IMO you could assume that an AMD GPU is available when someone uses this code.